backend/src/processStream.ts

```
@@ -155,70 +156,66 @@ interface MapField {
-        fs.unlink(jobFile.file, (e: any) => { if (e) log({unlinkInputProcessingError: e, jobId}) });
-    }, 1000);
-    await finishedAsync(writeStream);
-  } catch(e) {
-    throw(e);
-  }
+  // const inputHeaders: string[] = [];
+  // let outputHeaders: any;
+  const mapField:MapField = {};
+  const jobId = crypto.createHash('sha256').update(job.data.randomKey).digest('hex');
+
+  validFields.forEach(key => mapField[job.data[key] || key] = key );
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)